core/capabilities/transmission/transmission.go

```diff
@@ -49,9 +49,9 @@ func ExtractTransmissionConfig(config *values.Map) (TransmissionConfig, error) {
 // GetPeerIDToTransmissionDelay returns a map of PeerID to the time.Duration that the node with that PeerID should wait
 // before transmitting. If a node is not in the map, it should not transmit.  The sharedSecret is shared by nodes in the
 // same DON and used to generate a deterministic schedule for the transmission delays.
-func GetPeerIDToTransmissionDelay(donPeerIDs []ragep2ptypes.PeerID, sharedSecret [16]byte, transmissionID string, tc TransmissionConfig) (map[p2ptypes.PeerID]time.Duration, error) {
+func GetPeerIDToTransmissionDelay(donPeerIDs []ragep2ptypes.PeerID, transmissionID string, tc TransmissionConfig) (map[p2ptypes.PeerID]time.Duration, error) {
```

Comment:
@ettec One minor nit here: I worry that by requiring callers to set transmission id correctly (to the wflow ID + execution ID) that we'll be encouraging misuse.

IMO it's better to require workflowID + workflowExecutionID to be passed in explicitly as variables named that way and then have the function add it; that should avoid the possibility of misuse (+ is nicely self-documenting too)